spec/integrations/timestamps_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+require "acceptance_helper"
+
+describe "Logidze timestamps", :db do
+  include_context "cleanup migrations"
+
+  def self.setup_models(timestamp_column:)
```

Comment:
Let's convert this method into a shared context with a parameter to be more _RSpec way_.